docs/src/pages/premium-themes/onepirate/modules/components/TextField.js

```diff
@@ -62,17 +62,19 @@ const styles = theme => ({
 function TextField(props) {
   const {
     classes,
-    InputProps: {
```

Comment:
Causes bad transpilation reproducible with https://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=AQYw9gdgzgLsDeAoYLQBsCGUoFMoBplUBJCABwFcYAFAJzDKgC4EjV0tdmFgBLcqi1KUa9RgGFM2PMKr5gAOiWzRDKJM54A8jAAWOWsAC-wALwIjhdiiUKVdNTv202xsxaskBMADIYARjhoDoyeKBBgAEJgtAAmBu4AZhhouGHAULwAXjjuAOQAtjixvBQFeekAykE4IKqhrrZgegZEJuZkYlAA3EA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env&prettier=false&targets=Node-6.6&version=7.7.3&externalPlugins=

Might already be fixed in new babel-plugin-transform-rest-spread. Maybe we don't need the custom fork of rest-spread anymore with preset-module. Not sure if the forked plugin is even used. But this fixes the bug for now. Need to revisit before merge.

Reply:
Not reverting this until https://github.com/babel/babel/issues/10735 is resolved

Reply:
See https://github.com/babel/babel/issues/10735